articles/logs/streams/azure-event-grid.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Azure Event Grid Log Streams
+description: Learn how to create an event-driven workflow using Azure Event Grid and send your tenant logs anywhere within the Azure ecosystem.
+toc: false
+topics:
+ - logs
+ - streams
+ - event-streams
+contentType: how-to
+---
+
+# Integrate Azure Event Grid with Auth0
+
+Azure’s Event Grid is a serverless event bus that acts as an intermediary allowing you to send data from Auth0 into the Azure ecosystem.
+
+You can create an event-driven workflow using Event Grid to send your Auth0 tenant logs to the targets of your choice (e.g., Azure Functions, Event Hubs, Sentinel and Logic Apps).
+
+See the Auth0 event type codes for a [full list](https://auth0.com/docs/logs/references/log-event-type-codes) of the events that Auth0 supports.
+
+## Send events from Auth0 to Azure Event Grid
+
+To send Auth0 events to Azure, you will need:
+
+1. Set up an event source (in this case, this is Auth0).
+2. Set up an event handler, the app or service where the event will be sent.
+
+For more information about these concepts, [see Concepts in Azure Event Grid](https://docs.microsoft.com/en-us/azure/event-grid/concepts)
+
+### Set up an Auth0 event source
+
+Part of the integration process is to set Auth0 up for use as an event source (this step happens on your Dashboard).
+Log in to the Auth0 Dashboard.
+
+1. Navigate to Logs > Streams.
+2. Click + Create Stream.
+3. Select Azure Event Grid and enter a unique name for your new stream.
+4. Create the event source by providing your Azure Subscription ID, Azure Region and a Resource Group name. 
+5. Click Save.
+
+Go to the Azure Portal to complete the final steps of the integration.
+
+## Set up an event handler
+
+Go to your Azure subscription and spin up a service that is supported as an event handler (for a full list of all supported event handlers go to this [article](https://docs.microsoft.com/en-us/azure/event-grid/event-handlers)).
+
+### Enable Event Grid resource provider
+
+If you haven’t previously used Event Grid, you will need to register the Event Grid resource provider. If you've used Event Grid before, skip to the next section.
```

Comment:
This needs to come before the previous section (Set up an Auth0 event source), as we won't be able to create the partner topic if they don't have the event grid provider.

Reply:
Maybe we should indicate this somewhere on the UI too. Or at least in the Tutorial tab.

Reply:
Agreed. What do you think @camicano

Reply:
done